lesson_07/dags/sql/transfer_from_dwh_silver_to_dwh_gold.sql

```diff
@@ -0,0 +1,32 @@
+TRUNCATE TABLE  `{{ params.project_id }}.gold.user_profiles_enriched`
```

Comment:
1. An important note about file naming. `transfer_from_dwh_silver_to_dwh_gold.sql` - transfer.. what?? How can a user who's opening this file understand which data source this file is related to?

2. Real projects usually contain hundreds of sql files. I would recommend to put the files into dedicated to each datasource